packages/smart-forms-renderer/.storybook/test-runner.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('@storybook/test-runner').TestRunnerConfig} */
+module.exports = {
+  timeout: 15000,
```

Comment:
@Vlpros 15 is default timeout as I remember, do you think it's enough for all tests?